build.gradle

```diff
@@ -32,11 +33,16 @@ subprojects {
     apply plugin: 'com.palantir.baseline-eclipse'
     apply plugin: 'com.palantir.baseline-findbugs'
     apply plugin: 'com.palantir.baseline-idea'
+    apply plugin: 'com.palantir.docker-test-runner'
     apply plugin: 'org.inferred.processors'
 
     repositories {
         jcenter()
     }
 
     sourceCompatibility = 1.7
+
+    if (System.env.CIRCLE_TEST_REPORTS) {
+        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
```

Comment:
OK. There's no way to keep this configuration inside circle.yml, I guess? Would prefer to have-circle-specific things there if possible